src/vs/workbench/services/search/common/search.ts

```diff
@@ -186,20 +186,20 @@ export interface ITextSearchResultPreview {
 	cellFragment?: string;
 }
 
-export interface ITextSearchMatch {
-	uri?: URI;
+export interface ITextSearchMatch<U extends UriComponents = URI> {
```

Comment:
added a generic URI type to these, since the `INotebookCellMatch` uses it and has a raw URI version.